plans/GOAP_STATE.md

```
@@ -977,4 +977,23 @@ world_state:
+  # ═══════════════════════════════════════════════════════
+  memory_lifecycle_verification_completed: true
+  memory_lifecycle_via_turso_cli: true                  # sqld verified DB row counts + schema
+  memory_lifecycle_phase1_save: true                    # inject 2 concepts, associate, probe
+  memory_lifecycle_phase2_load_roundtrip: true           # export→import→probe yields identical results
+  memory_lifecycle_phase3_archive: true                 # archive marker concepts with metadata
+  memory_lifecycle_phase4_delete: true                  # delete + verify removed from active memory
+  memory_lifecycle_db_checksum: 821d5a78240b2aadf69a00ae805d11b2c1579f00377481461306e0430639daf4
+  memory_lifecycle_roundtrip_fidelity: true             # identical similarity scores (0.006055) across export/import
+  memory_lifecycle_metadata_preserved: true             # {"phase":"save","v":1} survives roundtrip
+  memory_lifecycle_followup_goap_created: true          # plans/GOAP_LIFECYCLE_VERIFICATION_FOLLOWUP.md
+  memory_lifecycle_sql_checks_fixed: true               # csm_ prefix + correct column names
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)